packages/core/useStepper/index.ts

```diff
@@ -0,0 +1,83 @@
+import { computed, ref } from 'vue-demi'
+
+export function useStepper<T extends string>(steps: readonly T[], initial?: T) {
+  const index = ref(steps.indexOf(initial ?? steps[0]))
+  const current = computed(() => steps[index.value])
+  const isFirst = computed(() => index.value === 0)
+  const isLast = computed(() => index.value === steps.length - 1)
+
+  function backTo(step: T) {
+    if (currentStepIsAfter(step))
+      goTo(step)
+  }
+
+  function goTo(step: T) {
+    index.value = steps.indexOf(step)
+  }
+
+  function next() {
+    if (isLast.value)
+      return
+
+    index.value++
+  }
+
+  function previous() {
+    if (isFirst.value)
+      return
+
+    index.value--
+  }
+
+  function isNext(step: T) {
+    return steps.indexOf(step) === index.value + 1
+  }
+
+  function isPrevious(step: T) {
+    return steps.indexOf(step) === index.value - 1
+  }
```

Comment:
If we provide `next` and `previous` as computed, I think we don't need this as ppl could just compare `step === next.value`

Reply:
If think `isNext('Step name')` is more practical than `next.value === 'Step name'`. I like the function syntax better.

I would rather keep that function, but if you really want it gone I'll remove it

Reply:
Sure. I am fine with this.